src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -50,6 +50,36 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
             return this;
         }
 
+        /// <summary>
+        /// Configures the public client application to use the recommended reply URI for the platform. 
+        /// See https://aka.ms/msal-net-default-reply-uri.
+        /// <list type="table">
+        /// <listheader>
+        /// <term>Platform</term>
+        /// <term>Default Reply URI</term>
+        /// </listheader>
+        /// <item>
+        /// <term>.NET desktop</term>
+        /// <term><c>https://login.microsoftonline.com/common/oauth2/nativeclient</c></term>
+        /// </item>
+        /// <item>
+        /// <term>UWP</term>
+        /// <term>value of <c>WebAuthenticationBroker.GetCurrentApplicationCallbackUri()</c></term>
+        /// </item>
+        /// <item>
+        /// <term>.NET Core</term>
+        /// <term><c>https://localhost</c></term>
+        /// </item>
```

Comment:
>// </item> [](start = 9, length = 10)

If we are mentioning specifics for netcore and netdesktop perhaps we should also mention specifics on the other platforms? @JM?

Reply:
Will be address in a later PR